README.adoc

```diff
@@ -1,48 +1,166 @@
-# Project Starfish 
+= STARFIX : Simplifying The Devflow
+:toc:
+:toc-placement!:
+endif::[]
 
-A  cross platform client side application that would let you open a file in IDE of your choice (vscode, eclipse, intellij, emacs, vi, etc.) and  other cammands locally directly from the browser or file system.This will enable an option to “Open in IDE” through browser extension on various websites like github, gitlab,gitbucket etc similar to one you see in this demo below.
+image:https://github.com/starfixdev/starfix/blob/master/starfix.png[Starfix Logo,title="Starfix Logo"]
```

Comment:
This urls looks wrong (no `images` folder) and  is it not sufficient to just have a relative link to the image ?
i.e. `image:images/starfix.png` ?